CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Enclosure
 - Enclosure Group
 - Uplink Sets
+- Storage Volume Attachment
```

Comment:
Please arrange the resources alphabetically

Reply:
Done